workers/unity/Assets/Playground/Scripts/Worker/WorkerConnectorBase.cs

```diff
@@ -54,6 +54,11 @@ public async Task Connect(string workerType, ILogDispatcher logger)
                         new LogEvent("Cancelling connection attempt")
                             .WithField("WorkerType", workerType)
                             .WithField("Reason", "Required worker failed to connect"));
+#if UNITY_EDITOR
```

Comment:
Is there a ticket for doing this properly? We've discussed several less hacky ways of doing this so it might be worth commenting on this to that effect.

Reply:
I'll make a ticket for this